core/kazoo_couch/src/kz_couch_view.erl

```diff
@@ -39,15 +39,21 @@ design_info(#server{}=Conn, DBName, Design) ->
     Db = kz_couch_util:get_db(Conn, DBName),
     do_get_design_info(Db, Design).
 
+-spec all_design_docs(kz_data:connection(), ne_binary()) ->
+                             {'ok', kz_json:objects()} |
+                             couchbeam_error().
 -spec all_design_docs(kz_data:connection(), ne_binary(), view_options()) ->
                              {'ok', kz_json:objects()} |
                              couchbeam_error().
+all_design_docs(#server{}=Conn, DBName) ->
+    all_design_docs(#server{}=Conn, DBName, []).
 all_design_docs(#server{}=Conn, DBName, Options) ->
     Db = kz_couch_util:get_db(Conn, DBName),
-    Filter = [{'startkey', <<"_design/">>}
-             ,{'endkey', <<"_design0">>}
-              | Options
-             ],
+    Filter = props:set_values([{'startkey', <<"_design/">>}
+                              ,{'endkey', <<"_design0">>}
+                              ]
+                             ,Options
+                             ),
```

Comment:
this change doesn't work. `props:set_values/2` will remove `single` value options like `include_docs`.
either revert here or support preservation of `single` value in `props:set_value/3` (preferred).
```
-spec set_values(kz_proplist(), kz_proplist()) -> kz_proplist().
set_values([], Props) -> Props;
set_values([{K, V}|KVs], Props) ->
    set_values(KVs, set_value(K, V, Props)).

-spec set_value(kz_proplist_key(), kz_proplist_value(), kz_proplist()) ->
                       kz_proplist().
set_value(K, V, Props) ->
    [{K, V} | [KV || {Key, _}=KV <- Props, K =/= Key]].
```